source/leuven/slurm-accounting.rst

```diff
@@ -3,58 +3,81 @@
 Slurm Accounting
 ========================
 
-The :ref:`wICE <wice_t2_leuven>` cluster uses the Slurm Account Manager (SAM) for credit accounting. We developed so called SAM-commands to mimic the behavior of previously used MAM-commands (from the MOAB account manager). From the user point of view it should mostly be enough to change the prefix ``mam`` in the accounting command into ``sam``.
-
-The accounting system is very similar to a regular bank. Individual users have accounts that will be charged for the jobs they run. However, the number of credits on such accounts is fairly small, so
-research projects will typically have one or more project accounts associated with them. Users that are project members can have their project-related jobs charged to such a project account. In this how-to,
-the technical aspects of accounting are explained.
+The :ref:`wICE <wice_t2_leuven>` cluster uses the Slurm Account Manager (SAM) 
+for credit accounting.
+The reason is that having a valid Slurm account with sufficient credits in it
+is mandatory for running jobs on KU Leuven Tier-2 machines. 
+We developed so called SAM-commands to mimic the behavior of previously used 
+MAM-commands (from the MOAB account manager). 
+From the user point of view it should mostly be enough to change the prefix ``mam`` 
+in the accounting command into ``sam``.
+
+The accounting system is very similar to a regular bank. 
+Individual users have accounts that will be charged for the jobs they run. 
+However, the number of credits on such accounts is fairly small, so research projects 
+will typically have one or more project accounts associated with them. 
+Users that are project members can have their project-related jobs charged to such a 
+project account. 
+In this how-to, the technical aspects of accounting are explained.
 
 Checking an account balance
 ---------------------------
 
-Since no calculations can be done without credits, it is quite useful to determine the amount of credits at your disposal. This can be done by executing the following command on any KU Leuven Tier-2 node::
+Since no calculations can be done without credits, it is quite useful to determine the 
```

Comment:
This is not true for certain partitions (e.g. the interactive partition on wICE, as well as the dedicated partitions on wICE and Genius).

Reply:
true. I changed to text to this:
"It is a good practice to check the balance of your credit project(s) from time to time."